main.cpp

```diff
@@ -7,7 +7,7 @@
 
 
 // Blinking rate in milliseconds
-#define BLINKING_RATE_MS     500
```

Comment:
The "_MS" in the macro name isn't necessary anymore, though it's not really a problem...